server/index.js

```diff
@@ -53,19 +53,64 @@ app.use(errorMiddleware);
 app.post('/api/favorites/:userId/:hotelId', (req, res, next) => {
   const hotelId = req.params.hotelId;
   const userId = req.params.userId;
+  let hotelName = ''
+        //gets name of hotel from ID
+  fetch(`https://hotels4.p.rapidapi.com/properties/get-details?id=${hotelId}&locale=en_US&currency=USD&checkOut=2020-01-15&adults1=1&checkIn=2020-01-08`, {
+    method: 'GET',
+    headers: {
+      'x-rapidapi-key': `${process.env.API_KEY}`,
+      'x-rapidapi-host': 'hotels4.p.rapidapi.com'
+     }
+   })
+    .then(response => response.json())
+    .then(data => {
```

Comment:
If possible, have the client send you the hotel name so you don't have to fetch it.

Reply:
cannot have client send hotel name via the fetch URL since some hotel names have spaces or dashes and it produces an error